### PR TITLE
[onnx] Also check validation error in model load

### PR DESCRIPTION
**Pull request by take-cheeze**
closes https://github.com/pfnet/pytorch-pfn-extras/issues/809

### CODE REVIEW

**Review by linshokaku — APPROVED** [review]

**Comment by linshokaku** [comment]
https://github.com/pfnet/pytorch-pfn-extras/actions/runs/8428677067/job/23081629257
The nightly cpu test is now failing and I would like to merge it.

**Comment by take-cheeze** [comment]
OK! Let's check re-run working: https://github.com/pfnet/pytorch-pfn-extras/actions/runs/8291701992?pr=810

File: pytorch_pfn_extras/onnx/load.py [hunk 1]
```diff
@@ -24,7 +24,7 @@ def load_model(
     """
     try:
         return onnx.load_model(f, format=format, load_external_data=load_external_data)
-    except OSError as e:  # The ONNX may contain stripped large tensors.
+    except (OSError, onnx.checker.ValidationError) as e:  # The ONNX may contain stripped large tensors.
         if (load_external_data
                 and json.loads(Path(e.filename).name)["type"] != "stripped"):
```
Comment (take-cheeze) [inline pytorch_pfn_extras/onnx/load.py:29]:
```suggestion
                and isinstance(e, OSError)
                and json.loads(Path(e.filename).name)["type"] != "stripped"):
```

**Review by linshokaku — APPROVED** [review]
LGTM! Thanks!

**Comment by risicle** [comment]
I guess this prevents stripped-model auto-fallback though.

**Pull request merged**